src/react-ui-basics/AutocompleteSelect.js

```diff
@@ -333,11 +341,13 @@ class AutocompleteSelect extends React.Component {
                                                   }}
                         />}
                         {!withFilter && inputLabel && <label>{inputLabel}</label>}
-                        {(mode === MODE_MULTIPLE_MINI_INLINE || mode === MODE_MULTIPLE || mode === MODE_MULTIPLE_AUTO || mode === MODE_INLINE_MULTIPLE) && list}
+                        {false && (mode === MODE_MULTIPLE_MINI_INLINE || mode === MODE_MULTIPLE || mode === MODE_MULTIPLE_AUTO || mode === MODE_INLINE_MULTIPLE) && list}
                     </div>
                 )}
-                {!(mode === MODE_MULTIPLE_MINI_INLINE || mode === MODE_MULTIPLE || mode === MODE_MULTIPLE_AUTO || mode === MODE_INLINE_MULTIPLE) && list}
+                {false && !(mode === MODE_MULTIPLE_MINI_INLINE || mode === MODE_MULTIPLE || mode === MODE_MULTIPLE_AUTO || mode === MODE_INLINE_MULTIPLE) && list}
             </div>
+              {list}
+            </>
```

Comment:
don't like the idea of rendering `list` outside of the component

Reply:
moved inside